modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -49,3 +49,22 @@ You can force the reconciliation by making any minor change to the DPA itself.
 ----
 $ oc -n openshift-adp get secret/oadp-<bsl_name>-<bsl_provider>-registry-secret -o json | jq -r '.data'
 ----
+
+
+[id="openshift-adp-controller-manager-seg-fault_{context}"]
+== `openshift-adp-controller` segmentation fault
+
+The `openshift-adp-controller-manager` pod fails with a crash loop segmentation fault, when the pod crashes and is restarted indefinitely until returning a segmentation fault, when configuring a DPA with `cloudstorage` and `restic` enabled.
+
+You can have either `velero` or `cloudstorage` defined, as they are mutually exclusive fields.
```

Comment:
I am having a hard time understanding the relationship between Restic, Velero, and CloudStorage. See my second suggestion above. 

Is Restic a part of Velero or CloudStorage? Or is Restic it an independent object fails because both CloudStorage and Velero are configured? Apologies if this distinction would be obvious to customers, but I am getting a little disoriented.

Reply:
@michaelryanpeter - Velero supports backing up and restoring Kubernetes volumes using a free open-source backup tool called [restic](https://github.com/restic/restic). This support is considered beta quality. Please see the list of [limitations](https://velero.io/docs/v1.5/restic/#limitations) to understand if it fits your use case.

Velero allows you to take snapshots of persistent volumes as part of your backups if you’re using one of the supported cloud providers’ block storage offerings (Amazon EBS Volumes, Azure Managed Disks, Google Persistent Disks). It also provides a plugin model that enables anyone to implement additional object and block storage backends, outside the main Velero repository.

The restic integration was added to give you an out-of-the-box solution for backing up and restoring almost any type of Kubernetes volume. This integration is an addition to Velero’s capabilities, not a replacement for existing functionality. If you’re running on AWS, and taking EBS snapshots as part of your regular Velero backups, there’s no need to switch to using restic. However, if you need a volume snapshot plugin for your storage platform, or if you’re using EFS, AzureFile, NFS, emptyDir, local, or any other volume type that doesn’t have a native snapshot concept, restic might be for you.

Restic is not tied to a specific storage platform, which means that this integration also paves the way for future work to enable cross-volume-type data migrations.